src/app/containers/Send/index.js

```diff
@@ -42,9 +66,19 @@ export default class Send extends Component {
       return
     }
 
-    var amounts = {}
-    amounts[this.type.value] = parseFloat(this.amount.value)
-    api.neonDB.doSendAsset(network.name, this.address.value, account.wif, amounts)
+    //Set Asset Type
+    let assetType
+    if (this.state.assetType === 0) {
+      assetType = 'NEO'
+    } else {
+      assetType = 'GAS'
+    }
+
+    let amounts = {}
+    amounts[assetType] = parseFloat(this.state.amount)
+    console.log('submit')
+    console.log(network.name, this.state.address, account.wif, amounts)
```

Comment:
Might be best to remove these if they were just for personal debugging

Reply:
Good find! Updated.